scripts/update/Updater.php

```diff
@@ -636,6 +636,6 @@ public function update($initialVersion)
             $this->setVersion('8.7.0');
         }
 
-        $this->skip('8.7.0', '8.7.1');
+        $this->skip('8.7.0', '8.8.0');
```

Comment:
The updating will require also updating the config with:
```php
'score_report_url' => array(
        'extension' => 'taoNccerDelivery',
        'controller' => 'ScoreReport',
        'action' => 'printing'
    )
```
Because in case it's not set we are returning this anyway.